config.toml

```diff
@@ -22,7 +22,7 @@ pygmentsStyle = "dracula"
       autoHeadingIDType = "github"
     [markup.goldmark.renderer]
       hardWraps = false
-      unsafe = false
+      unsafe = true
```

Comment:
Is this needed? We don't have much inline js or Html in our contents.

Reply:
Yes, `sig-list.md` uses it and it'll be our one way to make certain parts of the site look a bit better. =/